metricbeat/.gitignore

```diff
@@ -7,3 +7,4 @@ _meta/beat.full.yml
 /metricbeat.test
 /docs/html_docs
 
+target/
```

Comment:
In this case I would suggest to create a .gitignore file inside the dropwizard module.